charts/karmada/templates/_karmada_apiservice.tpl

```diff
@@ -32,6 +32,7 @@ metadata:
 spec:
   type: ExternalName
   externalName: {{ $name }}-aggregated-apiserver.{{ include "karmada.namespace" . }}.svc.{{ .Values.clusterDomain }}
+{{- if has "metricsAdapter" .Values.components }}
```

Comment:
I think it might be in the wrong position.

The outer condition is `{{- if eq .Values.installMode "host" }}`, then if my installMode is component, it will not run into this line?

Reply:
I think this block should be on the same level as the below block `{{- if has "search" .Values.components }}`

Reply:
Nice catch! Fixed.